src/Raven.Studio/typescript/viewmodels/database/tasks/editOlapEtlTask.ts

```diff
@@ -0,0 +1,583 @@
+                                   "saveEditedOlapTable",
+                                   "syntaxHelp",
+                                   "toggleAdvancedArea",
+                                   "deleteOlapTable",
+                                   "editOlapTable");
+
+        aceEditorBindingHandler.install();
+    }
+
+    activate(args: any) {
+        super.activate(args);
+        const deferred = $.Deferred<void>();
+
+        if (args.taskId) {
+            // 1. Editing an Existing task
+            this.isAddingNewOlapEtlTask(false);
+
+            getOngoingTaskInfoCommand.forOlapEtl(this.activeDatabase(), args.taskId)
+                .execute()
+                .done((result: Raven.Client.Documents.Operations.OngoingTasks.OngoingTaskOlapEtlDetails) => {
+                    this.editedOlapEtl(new ongoingTaskOlapEtlEditModel(result));
+                    this.showAdvancedOptions(!!result.Configuration.MaxNumberOfItemsInRowGroup);
+                    deferred.resolve();
+                })
+                .fail(() => {
+                    deferred.reject();
+                    router.navigate(appUrl.forOngoingTasks(this.activeDatabase()));
+                });
+        } else {
+            // 2. Creating a New task
+            this.isAddingNewOlapEtlTask(true);
+            this.editedOlapEtl(ongoingTaskOlapEtlEditModel.empty());
+            
+            this.editedTransformationScriptSandbox(ongoingTaskOlapEtlTransformationModel.empty());
+            this.editedOlapTableSandbox(ongoingTaskOlapEtlTableModel.empty());
+            
+            deferred.resolve();
+        }
+        
+        return $.when<any>(this.getAllConnectionStrings(), this.loadPossibleMentors(), this.loadServerSideConfiguration(), deferred)
+            .done(() => {
+                this.initObservables();
+            });
+    }
+
+    private loadPossibleMentors() {
+        return new getPossibleMentorsCommand(this.activeDatabase().name)
+            .execute()
+            .done(mentors => this.possibleMentors(mentors));
+    }
+    
+    compositionComplete() {
+        super.compositionComplete();
+
+        $('.edit-raven-olap-task [data-toggle="tooltip"]').tooltip();
+
+        popoverUtils.longWithHover($(".responsible-node"),
+            {
+                content: tasksCommonContent.responsibleNodeInfo
+            });
+
+        popoverUtils.longWithHover($(".keep-files-on-disk"),
+            {
+                content: `<small>Keep the output files on disk</small>`
+            });
+
+        popoverUtils.longWithHover($(".custom-prefix"),
+            {
+                content: `<small>Set custom prefix</small>`
+            });
+
+        popoverUtils.longWithHover($(".run-frequency"),
+            {
+                content: `<small>Frequency at which this task will be triggered</small>`
+            });
+
+        popoverUtils.longWithHover($(".max-items-in-row"),
```

Comment:
this is under advanced so may not be visible by default (please make sure tooltip is working when you defining new OLAP)

Reply:
removed